### ISSUE TITLE
Networkd: Document "FirewallMark=mark/mask" format?

### ISSUE THREAD

**Issue by iBug** [systemd/systemd#17587]
**Is your feature request related to a problem? Please describe.**

Not a problem but I'd like to see it implemented

**Describe the solution you'd like**

The `FirewallMark=` should support an extra syntax as follows:

```ini
[RoutingPolicyRule]
FirewallMark=0x1/0xff
# Or any accepted integer format, like
#FirewallMark=1/255

Table=example
```

The above example should produce a routing rule similar to

```shell
ip rule add fwmark 0x1/0xff lookup example
#           ^^^^^^^^^^^^^^^
```

The behavioral result should be that any packet marked with `--set-xmark 0x1/0xff` in the "mangle" iptables table be matched by this routing rule and go through the specified routing table.

**Describe alternatives you've considered**

Use whatever method available to call `ip rule` commands directly

**Comment by iBug** [comment]
Luckily, it seems to have been supported since a while ago. However, [the documentation](https://www.freedesktop.org/software/systemd/man/systemd.network.html#FirewallMark=) ([archive](https://web.archive.org/web/20201112153813/https://www.freedesktop.org/software/systemd/man/systemd.network.html#FirewallMark=)) doesn't reflect this. Should it be updated?

**Comment by yuwata** [comment]
Thanks. Fix is waiting in #17593.
